schema.json

```diff
@@ -5875,8 +5875,8 @@
         }
       }
     },
-    "http://adaptivecards.io/schemas/adaptive-card.json": {
-      "$ref": "http://adaptivecards.io/schemas/adaptive-card.json"
+    "https://adaptivecards.io/schemas/adaptive-card.json": {
```

Comment:
there was Peter's question whether this definition at the end is needed?
but we may discuss that also later as the http->https fix is more important

Reply:
Yes, it can be done the other way, will remove it